opencensus/src/main/java/com/example/opencensus/Quickstart.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.opencensus;
+
+// [START monitoring_opencensus_metrics_quickstart]
+
+import com.google.common.collect.Lists;
+
+import io.opencensus.exporter.stats.stackdriver.StackdriverStatsExporter;
+import io.opencensus.stats.Aggregation;
+import io.opencensus.stats.BucketBoundaries;
+import io.opencensus.stats.Measure.MeasureLong;
+import io.opencensus.stats.Stats;
+import io.opencensus.stats.StatsRecorder;
+import io.opencensus.stats.View;
+import io.opencensus.stats.View.Name;
+import io.opencensus.stats.ViewManager;
+
+import java.io.IOException;
+import java.util.Collections;
+import java.util.Random;
+
+public class Quickstart {
+  private static final int exportInterval = 60;
```

Comment:
[Constants](https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names) should be in capital case:
```suggestion
  private static final int EXPORT_INTERVAL = 60;
```

Reply:
Done